src/get_manifest.ts

```diff
@@ -119,7 +123,9 @@ async function readImportedManifestFile(filename: string) {
 
 if (import.meta.main) {
   const protocol = getProtocolInterface(Deno.args);
-  const generatedManifest = await createManifest(Deno.cwd());
+  const generatedManifest = await getManifest(Deno.cwd());
+  // validate functions referenced in the manifest
+  forEachValidatedManifestFunction(Deno.cwd(), generatedManifest);
```

Comment:
This is the key line that should address the [bug you identified in your comment](https://github.com/slackapi/deno-slack-hooks/pull/57#pullrequestreview-1353643289).